app/package.json

```diff
@@ -39,7 +39,8 @@
     "ms": "^2.0.0",
     "npm": "^4.6.1",
     "object-path": "^0.11.3",
-    "raven": "^0.12.1",
+    "raven": "^2.3.0",
```

Comment:
now we depend on both these, do we need both `raven` and `raven-js`? Afaik one is for the renderer (browser side) and the other for the main (node) side?

Reply:
Oh.. I was wondering about that. Let me double check (both worked). Checking now

Reply:
I **think*

Reply:
I **think** it's important to use both of them as they are tailored for their respective environments, like tracking clicked links in the DOM for the browser one while the node one is tracking more OS specific stuff.

Reply:
@albinekb I agree. Done :)